prime-dotnet-webapi/Configuration/Database/Seeded Tables/RemoteAccessTypeLicenseConfiguration.cs

```diff
@@ -0,0 +1,84 @@
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 5 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 6 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 8 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 9 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 10 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 12 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 13 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 14 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 15 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 17 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 18 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 19 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 47 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 51 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 59 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 66 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 67 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 87 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 88 },
+                    new RemoteAccessTypeLicense { RemoteAccessTypeCode = RemoteAccessTypeEnum.PrivateCommunityHealthPractice, LicenseCode = 89 },
+
+                    // FNHA
```

Comment:
Revise to "FNHA for select physician and nurse practitioner license classes"

Reply:
Comment added.